app/commands/assigning.rb

```diff
@@ -23,38 +25,112 @@ def stream_name
   class AssignToUser < Assigning::Command
     attribute :user_id, Types::Uuid
     attribute :user_name, Types::String
+    attribute :to_whom_id, Types::Uuid
+    attribute :to_whom_name, Types::String
+
+    def call
+      publish(AssignedToUser, data: to_hash)
+    end
+  end
+
+  class AssignToSelf < Assigning::Command
```

Comment:
Probably for later but it are you thinking that we might split these up into separate files - I think they may be easier to navigate. No need to do it now, thinking something like `assigning/AssignToSelf.rb` and so on

Reply:
Agree